### PR TITLE
Migrate/keys

### PR DESCRIPTION
**Pull request by mikiquantum**
Closes: #1043

### CODE REVIEW

File: migration/files/00Initial_test.go [hunk 1]
```diff
@@ -2,36 +2,22 @@ package files
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
 	"testing"
 
-	"github.com/centrifuge/go-centrifuge/utils"
+	"github.com/centrifuge/go-centrifuge/migration/utils"
```
Comment (vedhavyas) [inline migration/files/00Initial_test.go]: why a new package name, can we keep them as part of migrations package if no other package is using these ?

File: migration/utils/utils.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package migrationutils
+
+import (
+	"crypto/rand"
+	"os"
+	"path/filepath"
+)
+
+// RandomSlice returns a randomly filled byte array with length of given size
+func RandomSlice(size int) (out []byte) {
```
Comment (vedhavyas) [inline migration/utils/utils.go]: we do have another function in utils right ?

File: migration/files/00Initial_test.go (hunk 1 above)
Comment (mikiquantum) [reply migration/files/00Initial_test.go]: I wanted to have a folder division for the migration files vs the migration wrapper. I am less interested on the package naming though, I am open to move it to migration or migrationfiles.

File: migration/utils/utils.go (hunk 2 above)
Comment (mikiquantum) [reply migration/utils/utils.go]: Yes we do, the main reason to duplicate is that we do not want to add any dependencies if possible with the code base, as we could extend this package and make it more configurable and its own repository for go-(leveldb-otherdb) migrations

**Pull request merged**